tests/cases/fourslash/server/autoImportCrossProject_paths_toDist2.ts

```diff
@@ -0,0 +1,43 @@
+/// <reference path="../fourslash.ts" />
+
+// @Filename: /common/tsconfig.json
+//// {
+////   "compilerOptions": {
+////     "module": "commonjs",
+////     "outDir": "dist",
+////     "composite": true
+////   },
+////   "include": ["src"]
+//// }
+
+// @Filename: /common/src/MyModule.ts
+//// export function square(n: number) {
+////   return n * 2;
+//// }
+
+// @Filename: /web/tsconfig.json
+//// {
+////   "compilerOptions": {
+////     "module": "esnext",
+////     "moduleResolution": "node",
+////     "noEmit": true,
+////     "paths": {
+////       "@common/*": ["../common/dist/src/*"]
+////     }
```

Comment:
Offtopic question: I was actually exploring using exactly this structure in one of the projects. However, I only want this because Remix doesn't allow me to alias paths - but it can infer aliases from the `compilerOptions.paths`.

I wonder what are the pros/cons of this structure. I guess that in this test case here it might be required to use non-relative paths as `/common/` isn't actually symlinked as a monorepo package. However, when referenced projects **are symlinked** in node_modules, I expect this to be unnecessary and less performant. In the IDE this might be roughly the same thanks to "behind-the-scenes in-memory .d.ts generation process" but when running `tsc -b` this might not be able to actually leverage the generated `.d.ts` files as the `paths` map to `/src/`, right?